components/automate-ui/src/app/page-components/simple-line-graph/simple-line-graph.component.html

```diff
@@ -0,0 +1,12 @@
+<svg #svg [attr.viewBox]="viewBox" 
+    [ngStyle]="{'width': width}" 
+    (window:resize)="onResize()">
+
+
+    <defs>
+      <linearGradient id="chef-gradient-purple" x1="0%" y1="0%" x2="100%" y2="0%" gradientTransform="rotate(315)">
+        <stop id="stop1" offset="0%" />
+        <stop id="stop2" offset="100%" />
+      </linearGradient>
+    </defs>
```

Comment:
This is placed within the SVG so that we can reuse it as a gradient across any items needing to have our gradient